src/components/dashboard/Profile.js

```diff
@@ -104,7 +104,7 @@ class Profile extends React.Component {
   }
 
   render() {
-    console.log(this.state)
```

Comment:
weird, thought I had gotten rid of this before my push last night. Even ran a search for `console.log`s because I added so many. Not sure how this one snuck by me, especially when I remember deleting this one explicitly.